src/esn.inbox.libs/app/services/jmap-client-provider/jmap-client-provider.spec.js

```diff
@@ -49,23 +55,20 @@ describe('The jmapClientProvider service', function() {
   });
 
   it('should return a fulfilled promise if jwt generation succeed', function(done) {
-    angular.mock.module(function($provide) {
-      $provide.value('generateJwtToken', function() {
-        return $q.when('expected jwt');
-      });
-    });
     config['linagora.esn.unifiedinbox.api'] = 'expected jmap api';
     config['linagora.esn.unifiedinbox.downloadUrl'] = 'expected jmap downloadUrl';
+    tokenAPIMock.getWebToken = () => $q.when({ data: 'jwt' });
     injectServices.bind(this)();
 
-    jmapClientProvider.get().then(function(client) {
-      expect(client).to.be.an.instanceof(jmapDraft.Client);
-      expect(client.authToken).to.equal('Bearer expected jwt');
-      expect(client.apiUrl).to.equal('expected jmap api');
-      expect(client.downloadUrl).to.equal('expected jmap downloadUrl');
+    jmapClientProvider.get()
+      .then(function(client) {
+        expect(client).to.be.an.instanceof(jmapDraft.Client);
+        expect(client.authToken).to.equal('Bearer jwt');
+        expect(client.apiUrl).to.equal('expected jmap api');
+        expect(client.downloadUrl).to.equal('expected jmap downloadUrl');
 
-      done();
-    }, done.bind(null, 'should resolve'));
+        done();
```

Comment:
do we have spaces here ?

Reply:
yes, no ?

Reply:
to match the indentation level, the `done` should be at the same level of the `expect`

![image](https://user-images.githubusercontent.com/6764881/122414025-bdad1d00-cf7e-11eb-8d55-17112a73cfe3.png)